contracts/account/src/auth/passkey.rs

```diff
@@ -26,7 +26,7 @@ struct QueryRegisterResponse {
 #[cw_serde]
 struct QueryAuthenticateResponse {}
 
-pub fn register(deps: Deps, addr: Addr, rp: String, data: Binary) -> ContractResult<Binary> {
+pub fn register(deps: &Deps, addr: Addr, rp: String, data: Binary) -> ContractResult<Binary> {
```

Comment:
`Deps` and `InstantiateMsg` are passed by value and each have 48 bytes and 80 bytes respectively. The problem is that they aren't being consumed in their methods which means that the processor must often allocate an unnecessary amount of stack memory and registers.

Instead, they are now being passed as references which is only 8 bytes on a x86_64 system.